base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1860,5 +1860,20 @@
         </macros>
       </value>
     </entry>
+    <entry>
+      <key>CP2800</key>
+      <value>
+        <categories/>
+        <type>CP2800</type>
+        <path>cp2800.opi</path>
+        <description>The OPI for the CP2800</description>
+        <macros>
+          <macro>
+            <name>CP2800</name>
+            <description>The CP2800 PV prefix (e.g. CP2800_01)</description>
```

Comment:
Can you please add a default IOC value? Especially if there are only a few of these expected per instrument

Reply:
done